features/FxA-I-11/README.md

```diff
@@ -0,0 +1,128 @@
+# Match 1 - Control vs. No Show
+> Eye Password Experiment FXA-80
+
+Part of the [Eye Password Experiment](https://github.com/mozilla/fxa/blob/rfeeley/eye-password-emoji/features/FxA-80-eye-password-experiment/README.md)
+
+Idea in AHA: https://mozilla.aha.io/ideas/ideas/FXA-I-11
+
+## Outcomes
+
+The success of this feature is being able to answer the question:
+
+"Is the 'Show' password button useful in its current form?".
+"Does it reduce the number of errors the users make on the change password page?"
```

Comment:
Ooops. Sorry, I confused about the context of this. I was thinking about the outcome for the entire feature FXA-80. Each of these "matches" probably share a common outcome, problem statement, and assumptions, and the work to do here is to figure out the hypothesis for each match.